### PR TITLE
Add new localizations: RU, BY

### PR DESCRIPTION
**Pull request by ymakei**

### CODE REVIEW

**Comment by bastibe** [comment]
Thank you for your pull request! I like it!

Is this ready to be merged?

**Comment by ymakei** [comment]
> Thank you for your pull request! I like it!
> 
> Is this ready to be merged?

Yes, i made local patch to "~/.emacs.d/elpa/org-static-blog". And it works fine

File: org-static-blog.el [hunk 1]
```diff
@@ -131,31 +131,49 @@ The tags page lists all posts as headlines."
...
      ("en" . "%d %b %Y")
-     ("pl" . "%Y-%m-%d"))
+     ("pl" . "%Y-%m-%d")
+     ("ru" . "%d.%m.%Y")
+     ("by" . "%d.%m.%Y"))
     (tags
      ("en" . "Tags")
-     ("pl" . "Tagi"))
+     ("pl" . "Tagi")
+     ("ru" . "Ярлыки")
```
Comment (MunGell) [inline org-static-blog.el]: Although, "Ярлыки" is absolutely correct translation, in most cases "Теги" is used (a transcription of "Tags" into Russian).

E.g. WordPress translates "Tags" as both "Теги" and "Meтки" (a word that is closer to "Marks" in English).
Interestingly, "Ярлыки" in WordPress is used for URL slugs (as much as I can tell from their translation file).

File: org-static-blog.el [hunk 2]
```diff
@@ -131,31 +131,49 @@ The tags page lists all posts as headlines."
...
      ("en" . "Posts tagged")
-     ("pl" . "Wpisy z tagiem"))
+     ("pl" . "Wpisy z tagiem")
+     ("ru" . "Публикации с ярлыками")
+     ("by" . "Публікацыі"))
     (no-prev-post
      ("en" . "There is no previous post")
-     ("pl" . "Poprzedni wpis nie istnieje"))
+     ("pl" . "Poprzedni wpis nie istnieje")
+     ("ru" . "Нет предыдущей публикации")
```
Comment (MunGell) [inline org-static-blog.el]: An alternative translation could be:

"Предыдущая запись не найдена"

Just to keep it more grammatically correct (at the moment it is a direct translation from English)

File: org-static-blog.el [hunk 3]
```diff
@@ -131,31 +131,49 @@ The tags page lists all posts as headlines."
...
+     ("by" . "Няма папярэдняй публікацыі"))
     (no-next-post
      ("en" . "There is no next post")
-     ("pl" . "Następny wpis nie istnieje"))
+     ("pl" . "Następny wpis nie istnieje")
+     ("ru" . "Нет следующей публикации")
```
Comment (MunGell) [inline org-static-blog.el]: Similarly, "Следующая запись не найдена"

File: org-static-blog.el (hunk 2 above)
Comment (ymakei) [reply org-static-blog.el]: "Предыдущая запись не найдена" == "Previous record not found"

File: org-static-blog.el (hunk 3 above)
Comment (ymakei) [reply org-static-blog.el]: Next record not found

File: org-static-blog.el (hunk 1 above)
Comment (ymakei) [reply org-static-blog.el]: Just try this - https://translate.yandex.ru/?lang=en-ru&text=tags
Or this - https://www.translate.ru/

Wordpress is not the authority for me)) Russian is my native language

Comment (MunGell) [reply org-static-blog.el]: Спорить не буду, но давно уже не видел сайтов где теги назывались бы ярлыками :)
---
As I said, absolutely correct translation according to dictionaries, just not very popular these days :)

Comment (ymakei) [reply org-static-blog.el]: Yes, but it looks more like ¨навешивать ярлыки¨ and not ¨добавлять тэги¨
Тэги - это примитивное заимствование, а ярлык (бирка) более точно отражает суть действия, но если есть мнение, что следует следовать )) мейнстриму, то ок, можно заменить на "Тэги".

Comment (MunGell) [reply org-static-blog.el]: Not insisting on my translation, just a thing to consider.
If you strongly feel "Ярлыки" is the right way to go, I don't mind 👍 it.

**Comment by bastibe** [comment]
Cool to see collaboration here. Please notify once you are happy with the pull request and it can be merged.

**Comment by ymakei** [comment]
> Cool to see collaboration here. Please notify once you are happy with the pull request and it can be merged.

It wouldbe so cool, if i can just load package with patch from melpa!!!

**Comment by MunGell** [comment]
LGTM 👍

**Comment by bastibe** [comment]
Thank you very much, then!

Melpa will typically pick up on these changes within 24 hours.

**Pull request merged**